src/pkg/cli/client/byoc.go

```diff
@@ -801,11 +817,12 @@ func dnsSafe(fqn qualifiedName) string {
 	return strings.ReplaceAll(strings.ToLower(string(fqn)), ".", "-")
 }
 
-func (b *byocAws) Destroy(ctx context.Context) error {
-	if err := b.setUp(ctx); err != nil {
-		return err
-	}
+func (b *byocAws) TearDown(ctx context.Context) error {
+	// if err := b.setUp(ctx); err != nil {
+	// 	return err
+	// }
 	return b.driver.TearDown(ctx)
```

Comment:
Is it possible that we end up with service containers are still running, but the CFN stack is torn down?

Reply:
That's possible, and it's intentional. Nothing in the running service stack should depend on stuff in the CFN stack.